server/services/licenceSearchService.js

```diff
@@ -82,11 +125,33 @@ module.exports = function createLicenceSearchService(
     async getLicencesInStageCOM(username) {
       const systemToken = await signInService.getClientCredentialsTokens(username)
       const licencesWithCOM = await licenceClient.getLicencesInStage('PROCESSING_RO', systemToken)
-      const bookingIds = await licencesWithCOM.map((l) => l.booking_id)
+      const bookingIds = licencesWithCOM.map((l) => l.booking_id)
       const prisoners = await prisonerSearchApi(systemToken).getPrisoners(bookingIds)
       const licencesAcc = []
       const prisonerDecoratedLicences = licencesWithCOM.reduce(getPrisonerDecoratedLicences(prisoners), licencesAcc)
       return getlicencesCSV(prisonerDecoratedLicences)
     },
+
+    async getLicencesRequiringComAssignment(username, prisonId) {
+      const systemToken = await signInService.getClientCredentialsTokens(username)
+      const licencesInStageWithAddressOrCasLocation = await licenceClient.getLicencesInStageWithAddressOrCasLocation(
+        'ELIGIBILITY',
+        systemToken
+      )
+      const bookingIds = licencesInStageWithAddressOrCasLocation.map((l) => l.booking_id)
+      const prisoners = await prisonerSearchApi(systemToken).getPrisoners(bookingIds)
+      const prisonersFilteredByPrisonCloseToHdced = prisoners.filter(
+        (p) => p.prisonId === prisonId && isCloseToHdced(p)
```

Comment:
could you exclude `prisoner.status === 'INACTIVE OUT'` prisoners here too?

Reply:
Moved as suggested, thanks!